fixtures/wasm-arc-futures/README.md

```diff
@@ -0,0 +1,39 @@
+# A basic test for uniffi components
+
+This test covers async functions and methods. It also provides examples.
+
+## Run the tests
+
+Simply use `cargo`:
+
+```sh
+$ cargo test
+```
+
+It is possible to filter by test names, like `cargo test -- swift` to only run
+Swift's tests.
+
+## Run the examples
+
+At the time of writing, each `examples/*` directory has a `Makefile`. They are
+mostly designed for Unix-ish systems, sorry for that.
```

Comment:
I'm confused by this one, since I don't see a `Makefile` being checked in.

What do you think about keeping these tests in your repo?  I don't see other bindings generators getting any use out of it and I'd rather not add it to the list of crates we need to publish (in fact, I'm hoping to shrink that list very soon).

Reply:
Oh ha, that was some copy pasta. I can definitely review and re-word this, if you still want the test.